Resfeber/View Controllers/TripsViewController.swift

```diff
@@ -122,30 +138,28 @@ extension TripsViewController: UICollectionViewDelegateFlowLayout {
         let cellHeight: CGFloat = cellWidth * 0.64
         return CGSize(width: cellWidth, height: cellHeight)
     }
-    
+
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
         20
     }
 }
 
 // MARK: - Collection View Data Source
-
 extension TripsViewController: UICollectionViewDataSource {
     func collectionView(_: UICollectionView, numberOfItemsInSection _: Int) -> Int {
-        destinationController.searchDestinations.count
+        tripService.getTrips()?.count ?? 0
```

Comment:
Is the `getTrips` method fetching trips? It would probably be good to only fetch once on load, and then saving the trips into an array that you can access.